### DISCUSSION TITLE
WMS Extension

### DISCUSSION THREAD

**Discussion by mindless-bureaucrat** [developmentseed/titiler#571] [Feature Request]
Hey Vincent,

Long time fan, first time caller. 

I saw your presentation at FOSS4G on TiTiler and immediately went home to implement it. Loving it, great work!

Part of my stack includes an unfortunate legacy web map application that only reads WMS. To remedy this, I've created an additional endpoint to `/cog` and `/mosaic` that functions as a lightweight WMS endpoint. The endpoint (`/wms`) mainly wraps the `.part()` function in factory.

Sounds like WMS and older WMS formats are less than sexy these days. Is this WMS feature something you'd be open to integrating to the main library as a PR? Or would this be better suited as a standalone extension? Happy to go either way.

**Comment by vincentsarago** [comment]
That would be a great addition (as long as it doesn't had big dependencies) 
Feel free to start a PR and I can help if needed
